File: app/src/main/java/com/cyberIyke/allvideodowloader/helper/RenameVideoPref.java
package com.cyberIyke.allvideodowloader.helper;

import android.content.Context;
import android.content.SharedPreferences;

public class RenameVideoPref {
    public SharedPreferences.Editor editor;
    public SharedPreferences preferences;


    public RenameVideoPref(Context context) {
        preferences = context.getSharedPreferences("RenameVideoPref", 0);
        this.editor = preferences.edit();
    }

    public void setString(String key, String value) {
        this.editor.putString(key, value).commit();
    }

    public String getString(String key, String defValue) {
        return this.preferences.getString(key, defValue);
    }

}

File: app/src/main/java/com/cyberIyke/allvideodowloader/helper/RenameVideoPref.kt
package com.cyberIyke.allvideodowloader.helper

import android.content.Context
import android.content.SharedPreferences

class RenameVideoPref constructor(context: Context) {
    var editor: SharedPreferences.Editor
    var preferences: SharedPreferences

    init {
        preferences = context.getSharedPreferences("RenameVideoPref", 0)
        editor = preferences.edit()
    }

    fun setString(key: String?, value: String?) {
        editor.putString(key, value).commit()
    }

    fun getString(key: String?, defValue: String?): String? {
        return preferences.getString(key, defValue)
    }
}